contracts/governance/GovernorAlpha.sol

```diff
@@ -59,9 +59,15 @@ contract GovernorAlpha is SafeMath96 {
 	/// @notice Percentage of current total voting power require to vote.
 	uint96 public quorumPercentageVotes;
 
-	// @notice Majority percentage.
+	/// @notice Majority percentage.
 	uint96 public majorityPercentageVotes;
 
+	/// @notice Threshold for total favorVotes of the proposal to be able to cancelled by guardian (in percentage).
+	uint256 public constant totalVotesForCancellationThreshold = 80; // 80%
+
+	/// @notice Threshold for total participant of the proposal to be able to cancelled by guardian (in percentage).
+	uint256 public constant participantForCancellationThreshold = 80; // 80%
```

Comment:
pls mind code style convention - constants should be named with all capital letters with underscores separating words

Reply:
Fixed